app/src/main/res/layout/content_messages.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This attribute is duplicated and this won't let you run the app.. please make sure you can actually run the app before commiting

Reply:
okey